pages/orderSuccess.js

```diff
@@ -124,21 +124,9 @@ class OrderSuccessPage extends React.Component {
   }
 
   componentDidMount() {
-    const durationInSeconds = 10 * 1000;
-    const animationEnd = Date.now() + durationInSeconds;
-    const randomInRange = (min, max) => Math.random() * (max - min) + min;
-    const confettisParams = { startVelocity: 30, spread: 360, ticks: 60, zIndex: 0 };
-
-    const interval = setInterval(() => {
-      const timeLeft = animationEnd - Date.now();
-      if (timeLeft <= 0) {
-        return clearInterval(interval);
-      } else {
-        const particleCount = 50 * (timeLeft / durationInSeconds);
-        confetti({ ...confettisParams, particleCount, origin: { x: randomInRange(0, 0.3), y: Math.random() - 0.2 } });
-        confetti({ ...confettisParams, particleCount, origin: { x: randomInRange(0.7, 1), y: Math.random() - 0.2 } });
-      }
-    }, 250);
+    if (!this.props.data?.error) {
```

Comment:
Added this little check because I had the case a few weeks ago where the request failed and I ended up on an error page with confettis all over the place. It was pretty strange :smile:

Reply:
I love the idea of error pages with confetti. Something went wrong! But let's be happy , woo hoo!